test/tsconfig.json

```diff
@@ -6,6 +6,17 @@
     "files": [
         "utils/mocks.ts",
         "utils/MochaLoader.ts",
-        "AudioPlayr/_.ts"
+        "AudioPlayr/addEventImmediate.ts",
+        "AudioPlayr/addEventListener.ts",
+        "AudioPlayr/clearTheme.ts",
+        "AudioPlayr/pauseSound.ts",
+        "AudioPlayr/play.ts",
+        "AudioPlayr/playLocal.ts",
+        "AudioPlayr/playSound.ts",
+        "AudioPlayr/playTheme.ts",
+        "AudioPlayr/removeEventListeners.ts",
+        "AudioPlayr/setMutedOff.ts",
+        "AudioPlayr/setMutedOn.ts",
+        "AudioPlayr/soundStop.ts"
```

Comment:
No, I mean the public `pauseAll` method.

Reviewing the list of public methods, you're missing tests for `resumeAll`, `clearAll`, and `playThemePrefixed`. I would consider these lower priority, let's skip them for now and file an issue to add them later.